### PR TITLE
[Bug] URI read from file coded properly..

### PR DESCRIPTION
**Pull request by furszy**
Re coded the URI from file read. Can be easily tested parsing a file with the following text pattern:
`pivx:%address%?amount=%number%?label=%text%?message=%text%`

About the code that was here before, really no comments.. 

This solves #1103

### CODE REVIEW

**Comment by Fuzzbawls** [comment]
Looks like you're missing an include to `<QFile>`

**Comment by furszy** [comment]
yep, didn't check our friend travis.. seems that only linux needs it, the rest of the OS are fine. One second

File: src/qt/openuridialog.cpp [hunk 1]
```diff
@@ -63,6 +64,30 @@ void OpenURIDialog::on_selectFileButton_clicked()
...
+        return;
+    }
+
+    if (file.open(QIODevice::ReadOnly | QIODevice::Text)){
+        QByteArray r = file.readAll();
+        if (r.size() > 200) {
+            inform(tr("Parsed data too large"));
+            return;
+        }
+        QString str = QString::fromStdString(r.toStdString());
```
Comment (Fuzzbawls) [inline src/qt/openuridialog.cpp]: This is non-op on Qt versions prior to 5.4: (`error: ‘class QByteArray’ has no member named ‘toStdString’`). We currently support down to Qt 5.0.

While I think we should certainly look at bumping our minimum supported Qt version, and have discussed doing just that, it is not going to happen for 4.0.0.

Comment (furszy) [reply src/qt/openuridialog.cpp]: Ok, doing the changes. What an old and useless QT versions that we are currently supporting.

Comment (furszy) [reply src/qt/openuridialog.cpp]: Done.

**Pull request merged**